src/cloudWatchLogs/commands/copyLogStreamName.ts

```diff
@@ -20,7 +21,12 @@ export async function copyLogStreamName(uri?: vscode.Uri): Promise<void> {
             }
         }
         const parsedUri = parseCloudWatchLogsUri(uri)
-        await vscode.env.clipboard.writeText(parsedUri.streamName)
+        const parameters = parsedUri.parameters
+
+        if (!parameters.streamName) {
+            throw new Error()
```

Comment:
if this is actually possible, should probably 

1. log a message that mentions the `uri`, 
2. show a user message via `showViewLogsMessage()`

Reply:
Some sort of internal error is the only way I could imagine this error getting hit. However, in the case that happens an error message would be helpful, so I'll go ahead and add that.